interface/src/components/requests.js

```diff
@@ -25,49 +27,62 @@ export function getVisualData(files, type, username){
         formData.append('sites',files.sites,"sites.csv");
         formData.append('treatments',files.treatments,"treatments.csv");
         formData.append('experiments_treatments',files.experiments_treatments,"experiments_treatments.csv");
+        formData.append('shp_file',files.shapefile);
         formData.append("username", username);
         params= 'getTreatmentSites'
     }
     
-    let res= fetch(`http://localhost:3001/${params}`,  {
+    let res= fetch(`http://localhost:8008/${params}`,  {
         method: 'POST',
         body: formData,
         redirect: 'follow'
       })
-        .then(response => response.json())
-        .then(result => result)
-        .catch(error => {
+    .then(response => response.json())
+    .then(result => result)
+    .catch(error => {
         throw Error(error)
         });
     return Promise.resolve(res);
 }
 
+export function getUserData(apikey){
+    const formData = new FormData();
+    formData.append("apikey", apikey);
+    let res= fetch(`http://localhost:8008/getUser`,  {
+        method: 'POST',
+        body: formData,
+        redirect: 'follow'
+      })
+    .then(response => response.json())
+    .then(result => result.user)
+    .catch(error => {
+        throw Error(error)
+    });
+    return Promise.resolve(res);
+}
 
 export function getGeoJSON(file){
     const formData = new FormData();
+    formData.append('shp_file', file, file.name);
 
-    formData.append("shp_file", file);
-
-    let res= fetch("http://localhost:3001/getGeoJSON", {
+    let res= fetch("http://localhost:8008/getGeoJSON", {
```

Comment:
See comment in docker-compose file above for reference: this is an example of where having the host URI defined as an environment variable would be handy

Reply:
New issue #72 for that